### ISSUE TITLE
Add underline for inline links 

### ISSUE THREAD

**Issue by lboehling** [patternfly/patternfly#6900]
Per DPO accessibility discussion -- our inline links should have an underline on default, not just on hover. Our text decoration tokens should already be spec'd out this way, but wanted to open an issue for tracking!

**Comment by lboehling** [comment]
https://github.com/patternfly/patternfly/issues/6869 this should fix this!

**Comment by mcoker** [comment]
closed by https://github.com/patternfly/patternfly/pull/6908
